xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/AddressTest.java

```diff
@@ -0,0 +1,52 @@
+package org.xrpl.xrpl4j.codec.addresses;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+import org.xrpl.xrpl4j.model.transactions.Address;
+
+public class AddressTest {
```

Comment:
I think the reason this isn't getting picked up by codecov is because it's in the wrong module. These tests should go in the `xrpl4j-model` module because these don't test any classes in the address codec module.